.github/workflows/build_and_scan_images.yaml

```diff
@@ -0,0 +1,28 @@
+name: Build and scan OCI images
+
+on:
+  schedule:
+  # every day at 1:00AM UTC
+  - cron: '0 1 * * *'
+
+jobs:
+  build-scan:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Build and scan for vulnerabilities
+        run: |
+          cd image-definitions/
+          ./install-tools.sh
+          ./setup.sh .
+          ./check-updates.sh || true
+          ./apply-patches.sh
+          ./build-scan.sh
```

Comment:
This works, but might be easier to debug on failures if it is broken up a bit.  Maybe something like:
```suggestion
      - name: Install requirements for image building
        run: |
          ./install-tools.sh
        working-directory: image-definitions
      - name: Pull remote files and apply patches
        run: |
          setup.sh .
          ./check-updates.sh || true
        working-directory: image-definitions
      - name: Build and scan for vulnerabilities
        run: |
          ./apply-patches.sh
          ./build-scan.sh
        working-directory: image-definitions
```